libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -182,4 +204,91 @@ describe('ListComponent', () => {
       });
     });
   });
+
+  describe('virtual scroll', () => {
```

Comment:
Consider: 
`with virtual scroll` 

to make it read like: `ListComponent with virtual scroll`

Reply:
In general consider taking a second look at these tests considering if the blocks can perhaps be rephrased to sound more like sentences using the Given - When - Then formula :D

Reply:
Done 👍